requirements-dev.txt

```diff
@@ -13,4 +13,4 @@ wheel
 flask-jwt
 
 # install flasgger itself as editable
--e .
\ No newline at end of file
+-e .
```

Comment:
Looks like your editor added a `\n`.

Reply:
@javabrett yeah it seems so, if @rochacbruno you wan to have it remove I can do it.